src/number-formatting/short-format.js

```diff
@@ -14,10 +14,11 @@ import formatNumber from './format';
 /**
  * Converts from a number to a string like "1k" or "100m".
  * @param num
+ * @param precision
  * @param options
  * @returns {string} Returns 0 when dates are equal. -1 when date1 less than date2. 1 when date1 greater than date2.
  */
-function shortFormat(num, options) {
+function shortFormat(num, precision, options) {

```

Comment:
Add new parameter to JSDocs. Not only here but in other places where we added new argument.